src/topic/TopicMessageSubmitTransaction.js

```diff
@@ -6,6 +6,7 @@ import TopicId from "./TopicId.js";
 import * as utf8 from "../encoding/utf8.js";
 import TransactionId from "../transaction/TransactionId.js";
 import Timestamp from "../Timestamp.js";
+import { util } from "../util.js";
```

Comment:
After updating `util` to be collection of methods instead of a class.
```suggestion
import * as util from "../util.js";
```